.github/workflows/release.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
 
       - name: Basic integrity checks
         run: |
+          go mod tidy
```

Comment:
I guess you want to check for the git status after that.
Did you want to make sure that go.mod doesn't require updates?
Something like:
```
git status --porcelain || exit 1
```
but @philpennock would know better

Reply:
This should be:
```
go mod tidy -diff
```